app/scripts/background.js

```diff
@@ -1205,7 +1209,21 @@ function setupSentryGetStateGlobal(store) {
 
 async function initBackground() {
   await onInstall();
-  initialize().catch(log.error);
+  initialize()
+    .then(() => {
```

Comment:
Nit: In general we should prefer `await` and `try..catch` over `.then`/`.catch. The main reason being that stack traces we get from async code are more complete when we use `await` (at least on V8, which supports async stack traces). With `.then` the stack trace is cut off.